### PR TITLE
Updates deps notably envoy

### PR DESCRIPTION
**Pull request by codefromthecrypt**
envoy 1.23.1 fixes os/x

### CODE REVIEW

**Review by arkodg — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -4,25 +4,25 @@ module github.com/tetratelabs/func-e
...
+	github.com/lufia/plan9stats v0.0.0-20220913051719-115f729f3c8c // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/power-devops/perfstat v0.0.0-20220216144756-c35f1ee13d7c // indirect
 	github.com/russross/blackfriday/v2 v2.1.0 // indirect
 	github.com/tklauser/go-sysconf v0.3.10 // indirect
 	github.com/tklauser/numcpus v0.5.0 // indirect
 	github.com/xrash/smetrics v0.0.0-20201216005158-039620a65673 // indirect
 	github.com/yusufpapurcu/wmi v1.2.2 // indirect
-	golang.org/x/sys v0.0.0-20220520151302-bc2c85ada10a // indirect
+	golang.org/x/sys v0.1.0 // indirect
```
Comment (codefromthecrypt) [inline go.mod:26]: amazing to ever see a real version here!

**Comment by codefromthecrypt** [comment]
I don't have fast internet right now, but I'll try before later tonight or roll back the CLI lib change which may have affected the `man` output

**Comment by codefromthecrypt** [comment]
@arkodg can you help source someone to troubleshoot and fix TestFuncERun_StaticFilesystem? Seems envoy had some configuration drift perhaps.

**Comment by arkodg** [comment]
> @arkodg can you help source someone to troubleshoot and fix TestFuncERun_StaticFilesystem? Seems envoy had some configuration drift perhaps.

If this isn't pressing, I'll take a look at the test next week, and will attempt to fix it

**Comment by dio** [comment]
Sorry to chime in, following probably would help:

```diff
diff --git a/e2e/static-filesystem.yaml b/e2e/static-filesystem.yaml
index ac63369..40a6a02 100644
--- a/e2e/static-filesystem.yaml
+++ b/e2e/static-filesystem.yaml
@@ -33,4 +33,5 @@ static_resources:
                               filename: "response.txt"
                 http_filters:
                   - name: envoy.filters.http.router
-                    typed_config: { }
+                    typed_config:
+                      "@type": type.googleapis.com/envoy.extensions.filters.http.router.v3.Router

```

**Comment by codefromthecrypt** [comment]
thanks to @dio this is now ready to roll

**Pull request merged**